codility_training/lessons.lesson05.PrefixSums.GenomicRangeQuery/sjjyy.java

```diff
@@ -0,0 +1,96 @@
+package lessons.lesson05.PrefixSums.GenomicRangeQuery;
+
+
+class sjjyy
+{
+	// timeout error
+	public static int[] solution2(String S, int[] P, int[] Q)
+	{
+
+		int len = P.length;
+		int[] result = new int[len];
+		
+		for (int i = 0; i < len; i++)
+		{
+			String str = S.substring(P[i], Q[i] + 1);
+			
+			if (str.contains("A"))
```

Comment:
반복문을 끝까지 돌아서 그래요! 지금 코드에서 return만 반복문 안에서 하는 것으로 변경하면 된답니다~
```suggestion
			if (str.contains("A")) return 1
			if (str.contains("X")) return 2
			if (str.contains("G")) return 3
			if (str.contains("T")) return 4
```